active/teachingtip/TeachingTipSpec.md

```diff
@@ -0,0 +1,565 @@
+
+A teaching tip can be used without a pointer when the information that it is presenting is not specific to a UI element. 
+
+![A sample app with a teaching tip in the bottom right corner. The tip title reads "Saving automatically" and the subtitle reads "We save your changes as you go - so you never have to." There is a close button on the top right corner of the teaching tip.](TeachingTipSampleApp.jpg)
+
+A teaching tip can require manual acknowledgement before dismissing or a teaching tip may also be light-dismiss enabled so that it will dismiss when a user scrolls or interacts with other elements of the application.
+
+A teaching tip can require the user to dismiss it via an "X" button in a top corner or a "Close" button at the bottom. A teaching tip may also be light-dismiss enabled in which case there is no dismiss button and the teaching tip will instead dismiss when a user scrolls or interacts with other elements of the application. Because of this behavior, light-dismiss tips are the best solution when a tip needs to be placed in a scrollable area. 
+
+![A sample app with a light-dismiss teaching tip in the bottom right corner. The tip title reads "Saving automatically" and the subtitle reads "We save your changes as you go - so you never have to."](TeachingTipLightDismissSample.jpg)
+
+
+### Create a teaching tip
+
+Here's the XAML for a pointing teaching tip control that demonstrates the default look of the TeachingTip with a title and subtitle. 
+Note that the teaching tip can appear anywhere in the element tree. In this example below, it's located in a ResourceDictionary.
+
+XAML
+```XAML
+<Button x:Name="SaveButton" Content="Save">
+    <Button.Resources>
+        <muxc:TeachingTip x:Name="AutoSaveTip"
+            Target="{x:Bind SaveButton}"
+            Title="Save automatically"
+            Subtitle="When you save your file to OneDrive, we save your changes as you go - so you never have to.">
+        </muxc:TeachingTip>
+    </Button.Resources>
+</Button>
+```
+
+C#
+```C#
+public MainPage()
+{
+    this.InitializeComponent();
+
+    if(!HaveExplainedAutoSave())
+    {
+        AutoSaveTip.IsOpen = true;
+        SetHaveExplainedAutoSave();
+    }
+}
+```
+
+Here's the result when this Page containing the button and teaching tip shown:
+
+![A sample app with a teaching tip pointing at the save button. The tip title reads "Saving automatically" and the subtitle reads "We save your changes as you go - so you never have to." There is a close button on the top right corner of the teaching tip.](TeachingTipPointerSample.jpg)
+
+### Non-pointing tips
+
+Not all tips relate to an element onscreen. For these scenarios, do not set the Target property and the teaching tip will instead display relative to the edges of the XamlRoot. However, a teaching tip can have the pointer removed while retaining placement relative to a UI element by setting the PointerMode property to "Off". 
```

Comment:
What if I want the tip to appear in the corner of a container that doesn't fill the root window? This is a common layout for an app. We use it in the XAML Controls Gallery, for example. ![XAML Controls Gallery](https://github.com/Microsoft/Xaml-Controls-Gallery/raw/master/README_Images/XamlGalleryLanding.PNG) Observe that the list of samples does not fill the window. Suppose I want to put a tip in the upper left corner of the samples listview to say "Select a sample from this list by clicking on it."

I cannot use a null Target because that will go to the corner of the XAML root. But if I set `Target=MyPanel` and `PointerMode=Off`, the tip will appear *outside* the target. There are no "inside" alignments. Do I have to specify `TopEdgeAlignedLeft` and set a negative `TargetOffset.Bottom` equal to the `ActualHeight` of the TeachingTip? Or do I set `Target=HeroImage` and ask for `BottomEdgeAlignedLeft`? (Would that mess up accessibility, setting an artificial target?)